telemetry/definitions/commonDefinitions.json

```diff
@@ -336,6 +336,11 @@
             "allowedValues": ["primary", "localSecondary", "globalSecondary"],
             "description": "The type of index being hit for the query/scan operation"
         },
+        {
+            "name": "dynamoDbTarget",
+            "allowedValues": ["table", "tableProperties", "tableStream"],
+            "description": "The type of entity referenced by a metric or operation"
```

Comment:
To make it a little clearer, can there be a callout for `dynamodb` based entity in the description ?